x-govuk/components/secondary-navigation/_secondary-navigation.scss

```diff
@@ -0,0 +1,102 @@
+.x-govuk-secondary-navigation {
+  @include govuk-font(19);
+
+  @include govuk-media-query($from: tablet) {
+    display: flex;
+    overflow-x: auto;
+    padding-top: govuk-spacing(2);
+    padding-bottom: govuk-spacing(7);
+  }
+
+  @include govuk-media-query($until: tablet) {
+    display: block;
+  }
+}
+
+.x-govuk-secondary-navigation__list {
+  width: 100%;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+  @include govuk-media-query($from: tablet) {
+    display: flex;
+    margin: 0;
+    white-space: nowrap;
+
+    box-shadow: inset 0 -1px 0 $govuk-border-colour;
+  }
+
+  @include govuk-media-query($until: tablet) {
+    margin-bottom: govuk-spacing(4);
+  }
+}
+
+.x-govuk-secondary-navigation__list-item {
+  @include govuk-media-query($from: tablet) {
+    display: flex;
+
+    & + & {
+      margin-left: govuk-spacing(4);
+    }
+  }
+
+  a,
+  a:link,
+  a:visited {
+    background-color: inherit;
+    display: block;
+    text-decoration: none;
+
+    @include govuk-media-query($from: tablet) {
+      border-bottom: 5px solid transparent;
+      padding-bottom: govuk-spacing(3) - 5px; // Compensate for 5px border
+    }
+
+    @include govuk-media-query($until: tablet) {
+      background-color: inherit;
+      border-left: 5px solid transparent;
+      padding: govuk-spacing(2);
+    }
+  }
+
+  a:focus {
+    color: $govuk-focus-text-colour;
+    background-color: $govuk-focus-colour;
+    box-shadow: 0 -2px $govuk-focus-colour, 0 5px $govuk-focus-text-colour;
+    border-color: $govuk-focus-text-colour;
+    border-left-color: transparent;
+    position: relative;
+  }
+}
+
+.x-govuk-secondary-navigation__list-item--current {
+  a,
+  a:link,
+  a:visited {
+    border-color: $govuk-brand-colour;
```

Comment:
Should this be link colour like in the primary navigation component? Although I note there’s an open issue on https://github.com/govuk-one-login/service-header/issues/29 suggesting that the current item border uses the brand colour. 

It’s hard to know the intention for many things in the GOV.UK Design System given that the brand colour is also the same as the link colour. I think I’m inclined to go towards keeping this as using the brand colour, and updating the primary navigation to match.

Reply:
Agree, feels like this should be the brand colour.